insights/specs/insights_archive.py

```diff
@@ -104,6 +104,7 @@ class InsightsArchiveSpecs(Specs):
     ls_docker_volumes = simple_file("insights_commands/ls_-lanR_.var.lib.docker.volumes")
     ls_etc = simple_file("insights_commands/ls_-lanR_.etc")
     ls_ocp_cni_openshift_sdn = simple_file("insights_commands/ls_-l_.var.lib.cni.networks.openshift-sdn")
+    ls_origin_local_volumes_pods = simple_file("insights_commands/python_-m_insights.tools.cat_--no-header_ls_origin_local_volumes_pods")
```

Comment:
@shzhou12 this spec should be to reflect the mangled name of the simple command above.